website/src/site/styles/layout/content.scss

```diff
@@ -63,8 +46,30 @@ $transition-timing: 300ms;
 
   a.header-anchor {
     color: var(--font-color);
-    font-size: inherit;
-    text-decoration: none;
+  }
+}
+
+a.header-anchor {
+  font-size: inherit;
```

Comment:
I think that the colour should be inherit as well:

<img width="415" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/7fe9f634-cd74-4b6a-b5bc-ceb86309ff60">